misk-hibernate/src/main/kotlin/misk/hibernate/HibernateModule.kt

```diff
@@ -141,7 +141,7 @@ class HibernateModule(
       val startVitessServiceKey = StartVitessService::class.toKey(qualifier)
       multibind<Service>().to(startVitessServiceKey)
       bind(startVitessServiceKey).toProvider(Provider<StartVitessService> {
-        StartVitessService(environment = environment, config = config, qualifier = qualifier)
+        StartVitessService(environment = Environment.TESTING, config = config, qualifier = qualifier)
```

Comment:
why even read the env var?

Reply:
I was testing a thing and wanted travis to build it

Reply:
FYI travis will build your branches without you opening a PR